util/package.go

```diff
@@ -260,6 +261,13 @@ func (p *Package) Validate() error {
 		return fmt.Errorf("no description set")
 	}
 
+	if p.Requirement.Elasticsearch.Versions != "" {
+		_, err := semver.ParseRange(p.Requirement.Elasticsearch.Versions)
+		if err != nil {
+			return fmt.Errorf("invalid elasticsesrch versions: %s, %s", p.Requirement.Elasticsearch.Versions, err)
```

Comment:
@mtojek Completely missed that change. Typo here.

Could you leave a quick comments in PR's if you also changed the registry logic. Sometimes its hard to find in the massive diff and Github has the tendency to collapse all the things at the end.

Reply:
https://github.com/elastic/package-registry/pull/306

I apologize for the confusion. I will mention all PR changes next time.

Reply:
Don't worry, I should have reviewed more carefully ;-)